bot/exts/moderation/incidents.py

```diff
@@ -130,6 +138,39 @@ def has_signals(message: discord.Message) -> bool:
     return ALL_SIGNALS.issubset(own_reactions(message))
 
 
+async def make_message_link_embed(ctx: Context, message_link: str) -> discord.Embed:
+    """
+    Create an embed representation of discord message link contained in the incident report.
+
+    The Embed would contain the following information -->
+        Author: @Jason Terror ♦ (736234578745884682)
+        Channel: Special/#bot-commands (814190307980607493)
+        Content: This is a very important message!
+    """
+    embed = discord.Embed()
+
+    try:
+        message_convert_object = MessageConverter()
+        message = await message_convert_object.convert(ctx, message_link)
+
+    except Exception as e:
+        embed.title = f"{e}"
+        embed.colour = Colours.soft_red
```

Comment:
Is the error going to be posted to the channel? I'm not sure that's desired. I'd say it should log the error and return without creating an embed.

Reply:
> Is the error going to be posted to the channel? I'm not sure that's desired. I'd say it should log the error and return without creating an embed.

It would be posted along the other embeds with the webhook. I did this so it is easier to know what happened about that message link. I could just log it with a `log.info` and not create a embed for it

Reply:
Let's just leave it for now. We can revisit it if we decide it's not helpful.